### ISSUE TITLE
Generative Language Modeling for Automated Theorem Proving

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#114]
# TL;DR
They proposed GPT-f, which automatically proves theorems using a transformer. The proofs of theorems are defined as a language model such as [GOAL,(theorem),PROOFSTEP,…]. They have succeeded in shortening the theorem proofs with 23 theorems compared to existing ones.
![Proof Search Tree](https://user-images.githubusercontent.com/24524018/93408947-bf06c780-f8d0-11ea-9313-f9e148493fc2.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.03393

# Submission Dates(yyyy/mm/dd)
2020/09/07

# Authors and institutions
Stanislas Polu, Ilya Sutskever
- OpenAI

# Methods

# Results

# Comments
